apps/cookbook/src/app/showcase/checkbox-showcase/checkbox-showcase.component.html

```diff
@@ -41,12 +41,24 @@ <h2>States</h2>
     <cookbook-checkbox-states-example #checkboxStatesExample></cookbook-checkbox-states-example>
   </cookbook-example-viewer>
 
+  <h2>Sizes</h2>
+  <p>
+    The sizes a checkbox can have.
```

Comment:
Small nitpick but the sentence seems incomplete to me. Suggest `The sizes that a checkbox can have`, `Allowed sizes of a checkbox`, or maybe omitting altogether.

Reply:
Fixed